embedded/scripts/env.sh

```diff
@@ -0,0 +1,2 @@
+export RUSTFLAGS="-C link-arg=-Tlink.x"
+export CARGO_TARGET_THUMBV7M_NONE_EABI_RUNNER="qemu-system-arm -cpu cortex-m3 -machine mps2-an385 -m 1G -nographic -semihosting-config enable=on,target=native -kernel"
```

Comment:
`-m 1G` on my host complain saying maximum memory for this machine is 16Mib, removing the parameter seems to work fine

`qemu-system-arm: Invalid RAM size, should be 16 MiB`

Reply:
Curious what is your qemu and OS version?  does it work with `-m 16m`?

Reply:
I'm thinking of changing this to `-m 1m`, but it would be good to know the answer to the above to make sure it works in your env too.

Reply:
yeah, sorry. 

On macOS Catalina with QEMU emulator version 5.2.0 it works only with `-m 16m` or without the option.

It doesn't work with `-m 1m` or `-m 1G`

Reply:
OK.  Ubuntu 20.04 has qemu 4.2 and it also works without the flag.  So I think I'll remove the flag if there are any other changes, or wait to a followup PR.